src/headerr/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import { Link } from "react-router-dom";
+import { AppBar, Toolbar, Typography, Button, Box } from "@mui/material";
+import { useMetaMask } from "../contexts/MetaMaskContext";
+
+const Header = () => {
+  const { isMetamaskConnected, connectToMetamask, disconnectFromMetamask } =
+    useMetaMask();
+
+  const linkStyle = {
+    textDecoration: "none",
+    color: "white",
+    marginRight: "16px",
+  };
+
+  const toolbarStyle = {
+    display: "flex",
+    justifyContent: "space-between",
+    backgroundColor: "green",
+  };
+
+  const linksContainerStyle = {
+    display: "flex",
+    alignItems: "center",
+  };
+
+  return (
+    <AppBar position="static">
+      <Toolbar style={toolbarStyle}>
+        <div style={linksContainerStyle}>
+          <Link to="/" style={linkStyle}>
+            Home
+          </Link>
+          <Link to="/profile" style={linkStyle}>
+            Profile
+          </Link>
+        </div>
+        <Box>
+          {isMetamaskConnected ? (
+            <Button color="inherit" onClick={disconnectFromMetamask}>
+              Disconnect MetaMask
+            </Button>
+          ) : (
+            <Button color="inherit" onClick={connectToMetamask}>
+              Connect MetaMask
```

Comment:
![image](https://github.com/milos135/reactNFTWithMetamask/assets/85948457/a880ba0b-834b-46e2-8348-4af142451ba2)
when i click disconnect and refresh my page it says i am still connected